public/js/report.js

```diff
@@ -259,7 +259,13 @@ var Report = React.createClass({
     },
 
     atLeastTwoHits: function () {
-        return this.state.queries.some(query => query.hits.length > 1);
+        var hit_num = 0;
+        _.each(this.state.queries, (function (obj) {
+            if (obj.hits.length > 0) {
```

Comment:
The `if` statement here is unnecessary: if a query has 0 hits, you will just be adding 0 to `hit_num`, which is fine.